src/search/middleware/search.js

```diff
@@ -79,7 +79,7 @@ async function handleGetSearchResultsError(req, res, error, options) {
     const reports = FailBot.report(error, Object.assign({ url: req.url }, options))
     // It might be `undefined` if no backends are configured which
     // is likely when using production NODE_ENV on your laptop
-    // where you might not have a HATSTACK_URL configured.
+    // where you might not have a HAYSTACK_URL configured.
```

Comment:
Sometimes misspelled words are intentional, or things reference obscure things, but here, I think it's just a typo